boot.php

```diff
@@ -665,17 +665,18 @@ function __construct() {
 			$this->scheme = 'https';
 		}
 
-		if (x($_SERVER,'SERVER_NAME')) {
+		if (x($_SERVER, 'SERVER_NAME')) {
 			$this->hostname = $_SERVER['SERVER_NAME'];
 
-			if (x($_SERVER,'SERVER_PORT') && $_SERVER['SERVER_PORT'] != 80 && $_SERVER['SERVER_PORT'] != 443)
+			if (x($_SERVER, 'SERVER_PORT') && $_SERVER['SERVER_PORT'] != 80 && $_SERVER['SERVER_PORT'] != 443) {
 				$this->hostname .= ':' . $_SERVER['SERVER_PORT'];
 			}
 			/*
 			 * Figure out if we are running at the top of a domain
 			 * or in a sub-directory and adjust accordingly
 			 */
 
+			/// @TODO This kind of escaping breaks syntax-highlightning on CoolEdit (Midnight Commander)
```

Comment:
I feel like it should be a personal note since not everyone uses CoolEdit. It's not a comment that concerns the code itself, but how you personally see it. And you can open an issue about improving the code style in general.